timeline/views.py

```diff
@@ -84,6 +83,15 @@ def patch(self, request, *args, **kwargs):
             error_message = get_validation_error_detail(e)
             return Response({"error": error_message}, status=status.HTTP_404_NOT_FOUND)
         
+    @swagger_auto_schema(auto_schema=None)
+    def put(self, request, *args, **kwargs):
+        raise MethodNotAllowed('PUT', detail='This method is not allowed.')
+    
+    @swagger_auto_schema(auto_schema=None)
+    def delete(self, request, *args, **kwargs):
+        raise MethodNotAllowed('DELETE', detail='This method is not allowed.')
+    
+        
```

Comment:
Here, I don't think this is necessary considering you have explicitly defined the class as a subclass of `generics.APIView`.

Reply:
I defined it explicitly since it showed up on the swagger docs and could be misleading.

Reply:
Oh, okay then! Thanks for the clarification 👍